core/modules/cart/store/actions.ts

```diff
@@ -275,6 +275,13 @@ const actions: ActionTree<CartState, RootState> = {
             action1: { label: i18n.t('OK') }
           })
         }
+        if (result.status === 'not_enough_stock') {
+          rootStore.dispatch('notification/spawnNotification', {
+            type: 'error',
+            message: i18n.t("not enough in stock", {name:product.name, qty:result.qty}),
```

Comment:
Cool PR - thanks @collymore. The only change I’d request is to change this i18n message. The convention still is to use the Full English translations as keys. Please just rename it and it’s fine